### PR TITLE
SDAN-726. Add the media icons (video, audio) to the home page dashboard display, article preview and article view

### PR DESCRIPTION
**Pull request by rbi-aap**
Add the media icons (video, audio) to the home page dashboard display, article preview and article view.

Please review, thanks

### CODE REVIEW

File: assets/components/cards/render/CardFooter.jsx [hunk 1]
```diff
@@ -19,6 +21,12 @@ CardFooter.propTypes = {
     pictureAvailable: PropTypes.bool,
     source: PropTypes.string,
     versioncreated: PropTypes.string,
+    audioIf: PropTypes.array,
```
Comment (MarkLark86) [inline assets/components/cards/render/CardFooter.jsx]: What does `audioIf` and `videoIf` stand for? A more descriptive naming convention would help other developers know what it is used for, for example `pictureAvailable` tells us that this property defines if a picture is available or not which also tells us it's value is a boolean.

**Comment by MarkLark86** [comment]
@rbi-aap A more descriptive commit title would help describe what the change is, without looking up the ticket in Jira. This helps, for example, when searching for a specific change using the history of file changes and/or commit titles.

File: assets/components/cards/render/CardFooter.jsx [hunk 2]
```diff
@@ -2,9 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import CardMeta from './CardMeta';
 
-function CardFooter({wordCount, pictureAvailable, source, versioncreated}) {
+function CardFooter({wordCount, pictureAvailable, source, versioncreated, audioAvailable,videoAvailable }) {
```
Comment (marwoodandrew) [inline assets/components/cards/render/CardFooter.jsx]: Small thing style thing, maybe a space after the comma like the other parameters!

**Review by marwoodandrew — APPROVED** [review]

**Comment by MarkLark86** [comment]
@rbi-aap Could you please change the commit title to something more descriptive to define what is changed in this PR

**Pull request merged**